src/msw/textctrl.cpp

```diff
@@ -3177,6 +3259,61 @@ bool wxTextCtrl::GetStyle(long position, wxTextAttr& style)
     }
 #endif // wxUSE_RICHEDIT2
 
+    wxTextAttrUnderlineType underlineType = wxTEXT_ATTR_UNDERLINE_NONE;
+    switch ( cf.bUnderlineType )
+    {
+        case CFU_UNDERLINE:
+            underlineType = wxTEXT_ATTR_UNDERLINE_SOLID;
+            break;
+        case CFU_UNDERLINEDOUBLE:
+            underlineType = wxTEXT_ATTR_UNDERLINE_DOUBLE;
+            break;
+        case CFU_UNDERLINEWAVE:
+            underlineType = wxTEXT_ATTR_UNDERLINE_WAVE;
+            break;
+    }
+
+    wxColour underlineColour = wxNullColour;
```

Comment:
```suggestion
    wxColour underlineColour;
```
This is harmless but looks a bit jarring.